taskcluster/scripts/toolchain/android-gradle-dependencies.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+function get_abs_path {
+    local file_path="$1"
+    echo "$( cd "$(dirname "$file_path")" >/dev/null 2>&1 ; pwd -P )"
+}
+
+CURRENT_DIR="$(get_abs_path $0)"
+PROJECT_DIR="$(get_abs_path $CURRENT_DIR/../../../..)"
+
+pushd $PROJECT_DIR
+
+# We build everything to be sure to fetch all dependencies
+./gradlew --no-daemon assemble bundle test lint
```

Comment:
Be aware that "everything" has caused lots of issues in m-c automation.  It's easy to forget to add tasks, or to have variants that aren't exercised, etc, etc.  E.g., are you really sure you're getting all debug and release and Gecko variants here?  Maybe now, but how about later?

Make sure to publish/circulate this so that others know the expectation.

Reply:
+1

This may be missing at least `ktlint`, `detekt` and assembling the android device tests (`assembleAndroidTest`).

Reply:
Good point. I added `ktlint`, `detekt`, and `assembleAndroidTest`. I agree with you, we need to keep this list up to date.

How does mozilla-central keep up? I see https://searchfox.org/mozilla-central/rev/4e228dc5f594340d35da7453829ad9f3c3cb8b58/taskcluster/scripts/misc/android-gradle-dependencies.sh#14. Is this enough to not forget any future variants?